backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/TeamService.cs

```diff
@@ -113,15 +113,16 @@ public async Task UpdateTeamMemberRoleAsync(TeamMemberDto teamMemberDto)
         _context.TeamMembers.Update(teamMember);
         await _context.SaveChangesAsync();
     }
-
-
+    
     public async Task DeleteTeamAsync(long teamId)
     {
         var teamEntity = await GetTeamByIdAsync(teamId);
         var slots = await _context.AvailabilitySlots.Where(el => el.TeamId == teamId).ToListAsync();
         var slotMembers = slots.SelectMany(_ => _.SlotMembers).ToList();
+        var calendarsAccesses = await _context.CalendarVisibleForTeams.Where(el => el.TeamId == teamId).ToListAsync();
```

Comment:
You can declare this as predicate for RemoteRange, without data loading

Reply:
I thought I can change all those call to 'Include', and then we will have ask database for data only 1 time. Will be it okay ?